staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -49,26 +51,10 @@ import (
 var (
 	Scheme = runtime.NewScheme()
 	Codecs = serializer.NewCodecFactory(Scheme)
-
-	// if you modify this, make sure you update the crEncoder
```

Comment:
Without those types the APIserver fails to start with

```
F0226 12:30:45.132296   26169 main.go:38] unable to install api resources: unable to setup API 
    &{[apiextensions.k8s.io/v1beta1] map[v1beta1:map[customresourcedefinitions:0xc000376be8 customresourcedefinitions/status:0xc000376bf0]] v1 <nil> 0xc00052ad20 {0xc00052ad20 [{[application/json] application/json [json] true 0xc000537240 0xc000537280 []  {} 0xc000537240} {[application/yaml] application/yaml [yaml] true 0xc0005372c0 <nil> []  <nil> <nil>} {[application/vnd.kubernetes.protobuf] application/vnd.kubernetes.protobuf [pb] false 0xc000552190 <nil> []  {} 0xc0001f4ae0}] 0xc000427f80 [{application/json true 0xc000537240 0xc000537280 0xc0001f4b40} {application/yaml true 0xc0005372c0 <nil> <nil>} {application/vnd.kubernetes.protobuf false 0xc000552190 <nil> 0xc0001f4b70}] 0xc000537240} 0xc0000cc5c0}: 
    [error in registering resource: customresourcedefinitions, no kind "ExportOptions" is registered for version "v1" in scheme "k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go:52",
     error in registering resource: customresourcedefinitions/status, no kind "ExportOptions" is registered for version "v1" in scheme "k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go:52"]
```

Reply:
Fixed